### ISSUE TITLE
Mismatch between helm chart in configuration and launch

### ISSUE THREAD

**Issue by johnksv** [InseeFrLab/onyxia#623]
TL;DR:
Based on picture below: Onyxia-web will show configuration matching chart version 1.0.0-alpha4, but onyxia-api will launch version 0.0.9

**Expected:**
Configuration and launching of a service from onyxia web should be done no the same helm chart version.

**Actual:**
Onyxia web retrieves a list of all available versions for the charts (endpoint in onyxia-api: `{catalogId}/charts/{chartName}`) which includes all versions (stable, alpha and beta version), see picture 1. Onyxia-web then uses the first element in this list to specify configuration.

When launching the `packageVersion` is not included in the request to the api. Helm will therefore use the latest **stable** version when installing the chart.

**Suggested solution**
Onyxia web should include `packageVersion` in the request to onyxia api. Also it would be nice if one in "developer mode" could choose version, but in "non-developer-mode" will use latest stable version.

Picture 1:
<img width="643" alt="Screenshot 2023-10-12 at 08 18 34" src="https://github.com/InseeFrLab/onyxia/assets/10381866/b7a1f621-7069-4be6-8ce1-0115af40df4a">

**Comment by fcomte** [comment]
There were some effort on version management and indeed the idea was to let user chart version on onyxia-web.
Unfortunely this work was never merged in the main branch so I think the gap is now very important.

I think your short workaround (sending version)  is a good idea to have a more consistant behaviour asap.
I would prefer working on data explorer over a complete version system in Onyxia Helm component.

**Comment by johnksv** [comment]
Fixed in version 5.3.0, from what I can see

**Comment by garronej** [comment]
Again thanks @johnksv,  
Really apreciate you folloing up on the issues!
